### DISCUSSION TITLE
translation to czech

### DISCUSSION THREAD

**Discussion by justmishka** [OWASP/mastg#2045] [Ideas]
Hi,

I am student of the master degree and my master theses topic is Security of Mobile application.

I am using your checklist and I will translate it to the czech language for my school work. I was thinking that this might be usefull also to your repository? Because I dont see that the czech language is there :) 

Thank you for letting me know!

**Comment by cpholguera** [comment]
Hi @justmishka , we're very happy to hear that. Thanks for taking the initiative! The Checklists are generated automatically, to get a new one in your language please create the new language in the OWASP MASVS repo and we'll take care of the rest.

I've created an issue for you with more details here:
https://github.com/OWASP/owasp-masvs/issues/622

**Reply by cpholguera** [reply]
@justmishka please accept the invitation we've sent you so that we can add you to the team and assign you that issue. Thank you very much!
